Assets/HoloToolkit-Preview/QRTracker/Scripts/QRCode.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Collections;
+
+using System.Collections.Generic;
+using UnityEngine;
+
+#if WINDOWS_UWP
+
+using Windows.Perception.Spatial;
+
+#endif
+namespace HoloToolkit.Unity.QRTracking
+{
+    [RequireComponent(typeof(HoloToolkit.Unity.SpatialGraphCoordinateSystem))]
+    public class QRCode : MonoBehaviour
+    {
+        public QRCodesTrackerPlugin.QRCode qrCode;
+        private GameObject qrCodeCube;
+
+        public float PhysicalSize;
+        public string CodeText { get; set; }
+
+        private TextMesh QRID;
+        private TextMesh QRText;
+        private TextMesh QRVersion;
+        private TextMesh QRTimeStamp;
+        private TextMesh QRSize;
+        private GameObject QRInfo;
+
+        private long lastTimeStamp = 0;
+
+        // Use this for initialization
+        void Start()
+        {
+            PhysicalSize = 0.1f;
+            CodeText = "Dummy";
+            if (qrCode == null)
+            {
+                throw new System.Exception("QR Code Empty");
+            }
+
+            PhysicalSize = qrCode.PhysicalSizeMeters;
+            CodeText = qrCode.Code;
+
+            qrCodeCube = gameObject.transform.Find("Cube").gameObject;
+            QRInfo = gameObject.transform.Find("QRInfo").gameObject;
+            QRID = QRInfo.transform.Find("QRID").gameObject.GetComponent<TextMesh>();
+            QRText = QRInfo.transform.Find("QRText").gameObject.GetComponent<TextMesh>();
+            QRVersion = QRInfo.transform.Find("QRVersion").gameObject.GetComponent<TextMesh>();
+            QRTimeStamp = QRInfo.transform.Find("QRTimeStamp").gameObject.GetComponent<TextMesh>();
+            QRSize = QRInfo.transform.Find("QRSize").gameObject.GetComponent<TextMesh>();
+
+            QRID.text = qrCode.Id.ToString();
+            QRText.text = CodeText;
+            QRVersion.text = "Ver: " + qrCode.Version;
+            QRSize.text = "Size:" + qrCode.PhysicalSizeMeters.ToString("F04") + "m";
+            QRTimeStamp.text = "Time:" + qrCode.LastDetectedQPCTicks;
+            Debug.Log("Id= " + qrCode.Id + " PhysicalSize = " + PhysicalSize + " TimeStamp = " + qrCode.LastDetectedQPCTicks + " QRVersion = " + qrCode.Version + " QRData = " + CodeText);
+        }
+
+        void UpdatePropertiesDisplay()
+        {
+            // Update properties that change
+            if (qrCode != null && lastTimeStamp != qrCode.LastDetectedQPCTicks)
+            {
+                QRSize.text = "Size:" + qrCode.PhysicalSizeMeters.ToString("F04") + "m";
+#if true
```

Comment:
nit: I'd prefer a commented section of code over an always true/false pragma. At least the comment can say why it's not currently available